pdf/creator/basic_line.go

```diff
@@ -0,0 +1,88 @@
+package creator
```

Comment:
Add license header to top of each file
https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#8-license-header-in-each-go-file

Reply:
Actually redundant given next comment